bpf/kmesh/ads/include/cluster.h

```diff
@@ -11,6 +11,8 @@
 #include "endpoint/endpoint.pb-c.h"
 
 #define CLUSTER_NAME_MAX_LEN BPF_DATA_MAX_LEN
+#define MAGLEV_TABLE_SIZE    16381
```

Comment:
Why is table size set to this?

Reply:
The Maglev load balancer implements consistent hashing to upstream hosts. It uses the algorithm described in section 3.4 of [this paper](https://static.googleusercontent.com/media/research.google.com/en//pubs/archive/44824.pdf) with a fixed table size of 65537 (see section 5.3 of the same paper)，and currently set a default value MAGLEV_TABLE_SIZE, later  we can dynamic change this table size.